spec/features/idv/doc_auth/verify_info_step_spec.rb

```diff
@@ -181,13 +181,13 @@
       visit idv_verify_info_url
       expect(page).to have_current_path(idv_session_errors_failure_path)
 
-      travel_to(IdentityConfig.store.idv_attempt_window_in_hours.hours.from_now + 1) do
+      travel_to(IdentityConfig.store.idv_attempt_window_in_hours.hours.from_now + 2) do
         sign_in_and_2fa_user(user)
         complete_doc_auth_steps_before_verify_step
         complete_verify_step
 
         expect(page).to have_current_path(idv_phone_path)
-        expect(RateLimiter.new(user: user, rate_limit_type: :idv_resolution)).to be_limited
```

Comment:
I'm confused why we expected it to be limited, if it's outside of the window, it should have been not limited right?